default/scripting/ship_parts/Bays.focs.txt

```diff
@@ -9,4 +9,15 @@ Part
     location = OwnedBy empire = Source.Owner
     icon = "icons/ship_parts/fighters-1.png"
 
+Part
+    name = "FT_BAY_2"
+    description = "FT_BAY_1_DESC"
+    class = FighterBay
+    capacity = 3
```

Comment:
If it's just increasing the capacity by 1, should this really be a whole separate part, rather than an upgrade from a tech to the original bay part?